BESTPRACTICES.md

```diff
@@ -0,0 +1,1418 @@
+    <h3 className="title">Your Modal</h3> /* --- 2nd --- */
+    <button className="btn">Accept</button> /* --- 3rd --- */
+    <span className="close">X</span> /* --- 4th  --- */
+  </div>
+);
+```
+
+:white_check_mark: ***Do***
+
+```scss
+.modal {
+  // comes 1st...
+  height: 300px;
+  width: 400px;
+  z-index: 1;
+
+  .title {
+    // comes 2nd...
+  }
+
+  .btn {
+    // comes 3rd...
+  }
+
+  .close {
+    // comes 4th...
+  }
+}
+```
+
+:no_entry_sign: ***Don't***
+
+```scss
+.modal {
+  // comes 1st...
+  height: 300px;
+  width: 400px;
+  z-index: 1;
+
+  .btn {
+    // // comes 3rd...
+  }
+
+  .close {
+    // comes 4th...
+  }
+
+  .title {
+    // comes 2nd...
+  }
+}
+```
+
+Do not nest selectors more than three levels deep. When selectors become this long, you're likely writing CSS that is:
+
+* Strongly coupled to the HTML (fragile) **-- or --**
+* Overly specific **-- or --**
+* Not reusable
+
+:no_entry_sign: ***Don't***
+
+```scss
+.pageContainer {
+  .content {
+    .profile {
+      // STOP!
+    }
+  }
+}
+```
+
+## Variables
+
+* Begin a variable name with two dashes `--`.
+* Use dash-case CSS variable names (e.g. `$--font-regular`).
+* Do ***not*** use camelCase or snake_case variable names. 
+
+
+## Comments
+
+* Prefer line comments (`//` for Sass) to block comments.
+* Prefer comments on their own line. Avoid end-of-line comments.
+* Write detailed comments for code that isn't self-documenting:
+  - Uses of z-index
+  - Compatibility or browser-specific hacks
+
+# Other
+
+## Git
+
+#### Branch Types
+
+- The Daedalus `master` branch is protected by `develop`. Only `develop` is merged into `master`.
+- A release branch contains the version in its name in this format `release/version`. Example: (`release/0.12.0`).
+- Release branches are based from `master`, but their version specific changes are not merged into `master`.
+- To make additions to the Daedalus codebase, create a branch based from `develop`.
+- There are three main branch types --> `feature`, `chore`, and `fix`.
+- A `feature` branch adds a new feature to Daedalus that does not yet exist.
+- A `chore` branch is appropriate when cleaning up or refactoring an area within the codebase.
+- A `fix` branch is used to introduce a fix for a bug.
+
+#### Branch Prefix
+
+- A branch prefix is a combination of its type (`feature`, `chore`, or `fix`) and its YouTrack ID.
+- YouTrack ID's for Daedalus start with the letters `DDW-` + **issue number**.
+- A branch suffix is a concise description of the changes contained in the branch.
+
+*Examples*:
+
+* `feature/ddw-41-create-daedalus-best-practices-guide`
+* `chore/ddw-225-refactor-api-to-use-v1`
+* `fix/ddw-315-fix-wallet-stuck-on-syncing`
+
+#### Commits
+
+Commit messages should begin with the YouTrack issue ID.
+
+Example: `git commit -m "[DDW-41] Adds Git section to Best Practices"`
+
+## CHANGELOG
+
+* All Entries are in **ABC order**, written in **past tense**, and preferably **1 sentence**.
```

Comment:
This is the only part which we need to modify. Recently, we agreed to order the entries based on their creation date so that the newest entries are on top of each section.